spec/linter-spec.js

```diff
@@ -0,0 +1,93 @@
+'use babel';
+
+import PolymerIde from '../lib/polymer-ide';
+import temp from 'temp';
+import { Severity } from 'polymer-analyzer/lib/warning/warning';
```

Comment:
is there a better place to import this from or is this right? (sometimes we have a file further up the tree that exports many things)